src/huggingface/forte/huggingface/token_classification.py

```diff
@@ -0,0 +1,204 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Wrapper of the Token Classification models on HuggingFace platform (context
+understanding)
+"""
+import re
+from typing import Dict, Set, List, Optional, Tuple, Any
+from transformers import pipeline
+
+from forte.common import Resources
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.processors.base import PackProcessor
+from forte.common import ProcessorConfigError
+from forte.utils.tagging_scheme import bio_merge
+from forte.utils import get_class
+
+__all__ = [
+    "TokenClassification",
+]
+
+
+class TokenClassification(PackProcessor):
+    r"""Wrapper of the models on HuggingFace platform with pipeline tag of
+    `ner`.
+    https://huggingface.co/models?pipeline_tag=token-classification
+    This wrapper could take the models on HuggingFace platform with pipeline
+    tag of `ner` to make prediction on token classification tasks, for example,
+    NER, POS tagging, word segmentation, etc.
+    This processor takes user specified entry type in the input pack and the
+    prediction result goes to the user specified attribute name of that entry
+    type in the data pack. Please refer to the `default_config` function for
+    more information.
+
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.classifier = None
+
+    def initialize(self, resources: Resources, configs: Config):
+        super().initialize(resources, configs)
+        self.classifier = pipeline(
+            "ner",
+            model=self.configs.model_name,
+            tokenizer=self.configs.tokenizer,
+            framework=self.configs.framework,
+            device=self.configs.cuda_device,
+        )
+
+    def _process(self, input_pack: DataPack):
+        """Perform HuggingFace NER Pipeline on the input data pack.
+
+        Args:
+            input_pack: Input pack to fill
+        Returns:
+        """
+        if not self.configs.entry_type:
+            raise ProcessorConfigError("Please specify an input entry type!")
+
+        input_entry = get_class(self.configs.entry_type)
+        output_entry = get_class(self.configs.output_entry_type)
+
+        for entry_specified in input_pack.get(entry_type=input_entry):
```

Comment:
We don't need to get the class first, we can use the class name (str) to get entries.

```suggestion
        for entry_specified in input_pack.get(entry_type= self.configs.entry_type):
```